syft/distro/distro.go

```diff
@@ -8,23 +8,28 @@ import (
 
 // Distro represents a Linux Distribution.
 type Distro struct {
-	Type       Type
+	Type       string
```

Comment:
With `Type` being removed it feels wrong to keep this field named as `Type`.

Since we have a field called `IDLike` I would be in favor of renaming this `ID` so that we can get as close to the source naming of these fields as possible.

Reply:
I like this idea — the more we can reuse well-known nomenclature, the better.

We'd just need to be sure to understand the consequences downstream; that is, was there any existing logic that really did think of this as a "Type" for some reason? (This is an exported field that is used in several places, even just within Syft itself.)

Reply:
We can punt this to the next PR when @wagoodman is back.